docker/dex/config-local.yaml

```
@@ -0,0 +1,160 @@
+- id: secret
+  redirectURIs:
+  - http://localhost:8080/auth/callback
+  name: pdc
+  secret: secret
+
+# local users for testing
+# Enable local users
+# @doc https://dexidp.io/docs/connectors/local/
+enablePasswordDB: true
+staticPasswords:
+  - email: admin@example.com
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)